build.gradle.kts

```diff
@@ -109,6 +108,23 @@ dependencies {
   runtimeOnly("org.antlr:antlr4-runtime:$antlrVersion")
   antlr("org.antlr:antlr4:$antlrVersion")
 
+  intellijPlatform {
+    // Note that it is also possible to use local("...") to compile against a locally installed IDE
+    // E.g. local("/Users/{user}/Applications/IntelliJ IDEA Ultimate.app")
+    // Or something like: intellijIdeaUltimate(ideaVersion)
+    create(ideaType, ideaVersion)
+
+    pluginVerifier()
+    zipSigner()
+    instrumentationTools()
+
+    testFramework(TestFrameworkType.Platform.JUnit4)
+    testFramework(TestFrameworkType.Platform.JUnit5)
```

Comment:
It seems unexpected to require two frameworks, but it fails with only one, and runs with two, so...

Reply:
Fails how?

Reply:
Either unresolved items (the `@RunInEdt` annotation) or runtime failures, also about the wrong threading models. Is it expected to require two frameworks?

Reply:
It's unexpected. I'll take a look.

Reply:
Current state: IdeaVim uses JUnit 5 features, so requires the `JUnit5` test framework type. However, some standard test functionality, such as `EditorTestUtil` and `LightProjectDescriptor` are only defined in `JUnit4`, so it looks like both test frameworks are required, and any change would need to come from the platform.

Reply:
As above: Gradle plugin has renamed `TestFrameworkType.Platform.JUnit4` to `TestFrameworkType.Platform`, and we add `TestFrameworkType.JUnit5` as an additional dependency.